programs/bpf_loader/src/lib.rs

```diff
@@ -298,6 +298,20 @@ fn check_loader_id(id: &Pubkey) -> bool {
         || bpf_loader_upgradeable::check_id(id)
 }
 
+fn calculate_heap_cost(heap_size: u64, heap_cost: u64, enable_rounding_fix: bool) -> u64 {
+    const KILOBYTE: u64 = 1024;
```

Comment:
Nit: This is technically a [kibibyte](https://en.wikipedia.org/wiki/Kibibyte) not a [KB](https://en.wikipedia.org/wiki/Kilobyte). (see https://en.wikipedia.org/wiki/Orders_of_magnitude_(data))

Reply:
TIL kibibyte 👍🏼